_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -846,6 +846,13 @@ If you need to brush up on how the IaC Library works, read the
 link:https://gruntwork.io/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library/[How to use
 the Gruntwork Infrastructure as Code Library] guide.
 
+
+=== Prepare your `infrastructure-live` repository
+
+We've previously described exactly how to prepare your repository in the
+link:https://gruntwork.io/guides/foundations/how-to-configure-production-grade-aws-account-structure/#prepare-your-infrastructure-live-repository[Gruntwork Landing Zone guide].
+Follow the steps in that section to get your `infrastructure-live` repository set up for the next steps.
```

Comment:
NIT: should that logic be in https://gruntwork.io/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library/ instead of the LZ guide? This could be handled in a separate PR.

Reply:
Opting for a future PR.

Reply:
https://github.com/gruntwork-io/gruntwork-io.github.io/issues/449. Resolving this comment.